### DISCUSSION TITLE
Company Branding

### DISCUSSION THREAD

**Discussion by comaldave** [WealthInnovationsCA/WealthInnovationsCA-Hugo#3] [Q&A]
The Name of the Company is : Wealth Innovations Inc.
I have a slogan on the alpha version

The Branding needs to consists to consist of a round SVG logo on a round or square background That looks good from 16x to 420x and a Banner Which uses the LOGO with the Company Name and Slogan. Your color choices should be limited to 2 to 4 colors of three shades and gradients in any vector. The branding colors will need to tie in to the website itself.

**Comment by ClaireElyseCrawford** [comment]
Can you reassign me?

**Reply by comaldave** [reply]
I have copied your Wi logo from Facebook, the colour seems to be 1D 40 9A FF which is hsl(223°, 68%, 36%).
Monogram: WI
Company Name: Wealth Innovations Inc
Slogan: Your wealth, our innovation. Together we grow.
The theme colours should be the primary for the Logo
complementary is 180°, triad is ±120°, Analogous is ±30°, Compound or Split Complementary is ±150°.

The Secondary colour should be based upon one of the vectors of the colour scheme, the ancillary should be the alternate vector at 100% saturation, light and dark should be the two vectors at 80% and 10%  lightness respectively.

Another alternative is monochrome so 10%, 25%, 36%, 60%, 80% lightness.
